website/docs/schema-directives.mdx

```diff
@@ -59,7 +59,7 @@ GraphQL Tools provides convenient yet powerful tools for implementing directive
 Here is one possible implementation of the `@deprecated` directive we saw above:
 
 ```typescript
-import { mapSchema, getDirectives } from '@graphql-tools/utils';
+import { mapSchema, getDirective, MapperKind } from '@graphql-tools/utils';
```

Comment:
`getDirectives` was misspelled, and `MapperKind` not imported at all. This confused me when I first went through the documentation.